### PR TITLE
Mkdocs deplpoyment + styling

### PR DESCRIPTION
**Pull request by CBroz1**

### CODE REVIEW

**Review by kabilar — CHANGES_REQUESTED** [review]
Thanks @CBroz1.  Now that the initial docs pull request is merged, please pull from upstream.

File: docs/src/concepts.md [hunk 1]
```diff
@@ -36,10 +36,10 @@ These teams include:
...
 The primary acquisition systems are:
 
-+ Sutter
-+ Thorlabs
-+ Bruker
-+ Neurolabware
++ Sutter (we estimate 400 rigs in active use - TBC)
++ Thorlabs (we estimate 400 rigs in active use - TBC)
++ Bruker (we estimate 400 rigs in active use - TBC)
++ Neurolabware (we estimate 400 rigs in active use - TBC)
```
Comment (kabilar) [inline docs/src/concepts.md:42]:
```suggestion
+ Sutter
+ Thorlabs
+ Bruker
+ Neurolabware
```

**Review by kabilar — CHANGES_REQUESTED** [review]
Thanks @CBroz1.  Minor comment above.

**Comment by CBroz1** [comment]
Thanks @kabilar - Resolved suggested change

**Review by kabilar — APPROVED** [review]

**Pull request merged**